pkgs/applications/networking/mumble/default.nix

```diff
@@ -72,66 +49,63 @@ let
     type = "mumble";
 
     nativeBuildInputs = [ qt5.qttools ];
-    buildInputs = [ libopus libsndfile speex qt5.qtsvg rnnoise ]
-      ++ lib.optional stdenv.isLinux alsa-lib
+    buildInputs = [ flac libogg libopus libsndfile libvorbis qt5.qtsvg rnnoise speex ]
+      ++ lib.optional (!jackSupport) alsa-lib
       ++ lib.optional jackSupport libjack2
       ++ lib.optional speechdSupport speechd
-      ++ lib.optional pulseSupport libpulseaudio;
+      ++ lib.optional pulseSupport libpulseaudio
+      ++ lib.optional pipewireSupport pipewire;
 
     configureFlags = [
-      "CONFIG+=no-server"
-    ];
+      "-D server=OFF"
+      "-D bundled-celt=ON"
```

Comment:
Any reason we're using the bundled celt version?

Reply:
Answer from one of the developers:
> version 0.7 - everything else won't work
> If your repo's celt version matches this description, that should work as well, but if the user has a different version installed, that will cause issues
> Thus, I would probably recommend sticking with the bundled option for CELT

Also, looks like they made several patches for celt 0.7.0 https://github.com/mumble-voip/celt-0.7.0/commits/5a16cda6d78cda0cd14eb13c56c65d82724842e5

Reply:
Eh, can't wait for them to drop CELT altogether.